lib/src/maintenance.dart

```diff
@@ -206,6 +223,22 @@ Future<Maintenance> detectMaintenance(
         'Readme should inform others about your project, what it does, and how they can use it.',
         penalty: new Penalty(fraction: 500)));
   }
+  if (!exampleExists) {
+    final exampleDirExists = files.any((file) => file.startsWith('example/'));
+    if (exampleDirExists) {
+      maintenanceSuggestions.add(new Suggestion.hint(
+          'Maintain an example.',
+          'None of the files in your `example/` directory matches a known example patterns. '
```

Comment:
We could print out a full list of the supported example names, right?

Reply:
I wouldn't duplicate the list with the `lib/*` versions, and the `$package_example.dart` seemed a bit too long here. The others are listed here.